lib/protobuf/field/bool_field.rb

```diff
@@ -17,7 +17,15 @@ def self.default
       # #
 
       def acceptable?(val)
-        [true, false].include?(val)
+        [true, false].include?(val) || (val.is_a?(String) && %w(true false).include?(val))
```

Comment:
IMO the class checking is not necessary, you could remove that and everything else would work